base/cvd/cuttlefish/common/libs/utils/subprocess_managed_stdio.cc

```diff
@@ -140,4 +140,20 @@ int RunWithManagedStdio(Command&& cmd_tmp, const std::string* stdin_str,
   return code;
 }
 
+Result<std::string> CaptureSuccessfulStdout(Command command) {
+  std::string standard_out;
+  std::string standard_err;
+  int exit_code = RunWithManagedStdio(std::move(command), nullptr,
+                                      &standard_out, &standard_err);
+  CF_EXPECTF(exit_code == 0,
+             "Command failure: exit code = {}, stdout = '{}', stderr = '{}'",
+             exit_code, standard_out, standard_err);
+  return standard_out;
+}
+
+Result<void> RunsSuccessfully(Command command) {
+  CF_EXPECT(CaptureSuccessfulStdout(std::move(command)));
```

Comment:
Calling `CaptureSuccessulStdout` here means the command's stdout won't go to the parent's stdout like stderr presumably does.

Reply:
It'll look a bit strange at callsites, but it seems less confusing to drop this method entirely.

Callers could instead invoke `CaptureSuccessfulStdout` / `RunAndCaptureStdout` which has more obvious semantics from the name, and drop the captured output.